addons/hw_posbox_homepage/static/src/app/components/FooterButtons.js

```diff
@@ -0,0 +1,31 @@
+/* global owl */
+
+import useStore from "../hooks/useStore.js";
+import { CredentialDialog } from "./dialog/CredentialDialog.js";
+import { HandlerDialog } from "./dialog/HandlerDialog.js";
+import { RemoteDebugDialog } from "./dialog/RemoteDebugDialog.js";
+
+const { Component, xml } = owl;
+
+export class FooterButtons extends Component {
+    static props = {};
+    static components = {
+        RemoteDebugDialog,
+        HandlerDialog,
+        CredentialDialog,
+    };
+
+    setup() {
+        this.store = useStore();
+    }
+
+    static template = xml`
+    <div class="w-100 d-flex align-items-cente gap-2 justify-content-center">
+        <a class="btn btn-primary btn-sm" t-att-href="'http://' + this.store.base.ip + '/point_of_sale/display'" target="_blank">PoS Display</a>
+        <a class="btn btn-primary btn-sm" t-att-href="'http://' + this.store.base.ip + ':631'" target="_blank">Printer Server</a>
```

Comment:
can be displayed on "advanced mode" only IMHO it's too technical for average users

Reply:
+ should be invisible on Windows